app/scripts/lib/transaction/util.ts

```diff
@@ -145,6 +147,7 @@ async function addTransactionWithController(
     transactionParams,
     networkClientId,
   } = request;
+  console.log('tx addTransaction', {request})
```

Comment:
Useful for verifying that the correct `networkClientId` and `chainId` are being passed to `TransactionController.addTransaction()`